CHANGELOG.md

```diff
@@ -11,12 +11,21 @@ Deprecations and compatibility notes:
 - Added warning that ``unary_union`` will return ``'GEOMETRYCOLLECTION EMPTY'`` instead
   of None for all-None GeoSeries. (#2618)
 
+
 Bug fixes:
 
 - Ensure that GeoDataFrame created from DataFrame is a copy, not a view (#2667)
 - Fix mismatch between geometries and colors in ``plot()`` if an empty or missing
   geometry is present (#2224)
 
+Notes on (optional) dependencies:
```

Comment:
Shouldn't this be merged with `Deprecations and compatibility notes` above?

Reply:
I don't mind, was just being consistent with previous changelogs which all split this out separately.

Reply:
okay